cloud-tasks/app/package.json

```diff
@@ -8,7 +8,7 @@
   },
   "scripts": {
     "start": "node index.js",
-    "test": "mocha --timeout 10000"
+    "test": "mocha test/*.test.js --exit"
```

Comment:
What problem does `--exit` solve, also wondering if we should keep the longer timeout?